lib/tests/streamlit/runtime/metrics_util_test.py

```diff
@@ -246,6 +245,7 @@ def test_public_api_commands(self):
         """All commands of the public API should be tracked with the correct name."""
         # Some commands are currently not tracked for various reasons:
         ignored_commands = {
+            "experimental_connection",
```

Comment:
The reason why we add it here is that in the case of `experimental_connection` calling it without parameters will throw an exception before the `gather_metrics` decorator is executed or?

Reply:
Ah, yeah that's correct. The issue is that we've attached the `@gather_metrics` decorator to a helper function so that we can be sure that the `type=` kwarg has been converted to a class when it gets called, so the exception occurs before the decorator is executed.

I'll add a comment to explain what's going on here and just push it straight to the feature branch